src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +234,21 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad wallet with files under wallets/foo/:\n"
```

Comment:
```suggestion
                    "\nLoad wallet from the wallet dir:\n"
```
this is easier to understand imo, and is correct for default cases and when -walletdir is specified

Reply:
Done